src/packaging/discovery-engine.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=knoxAutoPolicy
+
+[Service]
+User=root
```

Comment:
Do we really need root user?

Reply:
We do not need this to run as super user/root user.